libretroshare/src/gxs/rsgds.h

```diff
@@ -51,8 +51,8 @@ class RsGxsSearchModule  {
 class MsgLocMetaData {
 
 public:
-    MsgLocMetaData(const MsgLocMetaData& meta){ msgId = meta.msgId; val = meta.val;}
-    MsgLocMetaData() {}
+    MsgLocMetaData(const MsgLocMetaData& meta): msgId(meta.msgId), val(meta.val) {}
+    MsgLocMetaData() = default;

```

Comment:
Now that you touch it, you should made this class a struct, remove public keyword, and this line too (the class have no "special" members like pointers so the default costructor would be generated anyway AFAIR )